optimum/exporters/executorch/recipes/cuda.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2025 The HuggingFace Team. All rights reserved.
```

Comment:
You are the author, so add Meta copyright.

Reply:
I think Hugging Face requires us to add this license.

Reply:
I think you're mixing two things: 

- License -- in this case Apache license, which I agree HF requires us. 
- Copyright header -- they don't have such requirement. See an example here: https://github.com/huggingface/optimum-executorch/blob/dfa2d7ab0bce75d865e923717072b645ed0432b1/optimum/executorch/attentions/custom_kv_cache.py#L1